eva-lib/src/test/resources/eva.properties

```diff
@@ -7,6 +7,7 @@ eva.mongo.read-preference=
 
 eva.mongo.collections.variants=vars
 eva.mongo.collections.files=files
+eva.mongo.collections.annotation_metadata=annotation_metadata
```

Comment:
even though they can be different, I would try to homogenize this property names and the ones in the pipeline, @cyenyxe what do you think? I think these are better than the ones in the pipeline ("db.collections.annotation.metadata.name") if we ever use other db backend, and the ".name" feels a bit redundant. Also, I don't know why I split "annotation" and "metadata" with a dot (it's not a different level). Not sure about the underscore case, though.

If we agree to use a common one, as starting point for the discussion, I suggest "eva.mongodb.collections.annotationMetadata"